automated/lib/android-test-lib

```diff
@@ -62,7 +62,11 @@ adb_root() {
     if [ "$(adb shell whoami)" = "root" ]; then
         echo "DUT already has adbd running as root"
     else
+        echo "lsusb -v before adb root"
+        lsusb -v
         adb root || info_msg "adb root timed out"
+        echo "lsusb -v after adb root"
```

Comment:
is adb root a blocking call? If not, this might be a bit too fast. We have 'wait-for-device' in the next line which suggests 'adb root' isn't blocking.

Reply:
adb root should not be blocking, but it seems caused the device number changes in the output of lsusb -v, so I added this debug for check.

Reply:
if you're too quick, the device won't show up in lsusb -v. IIRC 'adb root' causes adb disconnect and reconnect, so the enumeration change is expected.

Reply:
adb root won't be blocking , but it seems cause the device number in the output of lsusb -v changed, so I added it here for debug.
regarding to putting it before wait-for-device, my concern is that if the wait-for-device timed out, then lsusb won't get executed so I put it just before the wait-for-device command, but as your concern on the "too fast" problem, it's possible. let me add sleep 10 before running the lsusb -v command.